### PR TITLE
net-fs/nfs-utils: move to 1.3.2

### PR DESCRIPTION
**Pull request by mischief**
nfs-utils 1.2.9 is affected by undefined behavior that produces crashes
under our new gcc 4.9 compiler.

fixes coreos/bugs#468

### CODE REVIEW

**Comment by mischief** [comment]
**/!\ ACHTUNG! /!\**

the nfs systemd unit names have changed! they were moved from the gentoo portage repo into upstream nfs-utils.

should we override the upstream nfs-utils unit names so they maintain compatibility with gentoo's unit names?

**Comment by mischief** [comment]
i've made an additional commit to make symlinks to keep compatibility with the old unit names from the gentoo ebuild.

File: net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild [hunk 1]
```diff
@@ -12,7 +12,7 @@ SRC_URI="mirror://sourceforge/nfs/${P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="alpha amd64 arm arm64 hppa ia64 ~mips ppc ppc64 s390 sh sparc x86"
+KEYWORDS="~alpha amd64 arm ~arm64 hppa ~ia64 ~mips ppc ppc64 ~s390 ~sh sparc x86"
```
Comment (marineam) [inline net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild]: should arm64 be stable too?

Comment (mischief) [inline net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild]: arm64 is using 1.3.2.. so, i am not sure?

Comment (mischief) [inline net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild]: i lied, geoff added nfs-utils-1.3.2 but didn't mark it stable for arm64.

Comment (marineam) [inline net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild]: Hm, well it seems a bit odd to have both 1.3.1 and 1.3.2 ebuilds in the overlay. Or if they are both there both should be in sync and have the same fixes (symlinks etc) in them. Either way the arm64 keywords can remain unstable for now.

**Comment by mischief** [comment]
@marineam PTAL at the latest commit, which enabled rpcbind.service for rpcbind.target.

File: net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild [hunk 2]
```diff
@@ -109,13 +112,20 @@ src_install() {
 		doins id_resolver.conf
 	fi
 
-	systemd_dotmpfilesd "${FILESDIR}"/nfs-utils.conf
```
Comment (marineam) [inline net-fs/nfs-utils/nfs-utils-1.3.1-r5.ebuild]: The tmpfiles rules have been dropped, are the dirs now created by the service units or what?

**Comment by mischief** [comment]
@marineam re-added tmpfiles snippet, ptal

**Comment by mischief** [comment]
@marineam moved to nfs-utils 1.3.2, kola tests passing :-) ptal

File: net-fs/nfs-utils/nfs-utils-1.3.2-r8.ebuild [hunk 3]
```diff
@@ -123,10 +124,14 @@ src_install() {
 	fi
 	rm "${D}$(systemd_get_unitdir)"/nfs-config.service || die
 	sed -i -r \
-		-e "/^EnvironmentFile=/s:=.*:=${EPREFIX}/etc/conf.d/nfs:" \
```
Comment (marineam) [inline net-fs/nfs-utils/nfs-utils-1.3.2-r8.ebuild]: should this change to `"/^EnvironmentFile=/d"`?

Comment (mischief) [inline net-fs/nfs-utils/nfs-utils-1.3.2-r8.ebuild]: yes, that would get rid of `EnvironmentFile=-/run/sysconfig/nfs-utils`

**Comment by mischief** [comment]
@marineam fixed the sed, ptal

**Comment by marineam** [comment]
LGTM

**Pull request merged**